requirements.txt

```diff
@@ -0,0 +1,2 @@
+# These requirements are for DEVELOPMENT ONLY!
+-r dev-requirements.txt
```

Comment:
I would add something like `--editable .` This will allow installing RHO on local venv so we can develop and immediately test the changes.

More about that see https://caremad.io/2013/07/setup-vs-requirement/